index.html

```diff
@@ -9082,6 +9090,45 @@ <h2>Screen capture</h2>
  <li><p>Return <a>success</a> with <var>canvas</var>.
 </ol>
 
+<p>In order to <dfn>draw a scrollable region from the framebuffer</dfn>,
```

Comment:
What exactly is that part about? Is it for the document element, or a child node? For child nodes we have `Take Element Screenshot`. Otherwise check you do below we already have at the top of the former list.

I feel we don't have to repeat all the steps again.

Reply:
This is for describing taking the screenshot for the full page instead of only the visible viewport.
It's for document.scrollingElement.
I did not use "draw a bounding box from the framebuffer" because I think step 2 and step 3
do not satisfy the requirement for full page screenshot.

"2. Let paint width be the initial viewport’s width – min(rectangle x coordinate, rectangle x coordinate + rectangle width dimension).
3. Let paint height be the initial viewport’s height – min(rectangle y coordinate, rectangle y coordinate + rectangle height dimension)."

Notice the - denotes the minus sign, so I think  for Take Screenshot and Take Element Screenshot, the width and height will not be greater than the visible viewport's width and height but the full page screenshot takes the document.scrollingElement.scrollWidth and scrollHeight which can be greater. So I removed step 2 and 3 and uses the scrollWidth and Height directly for paint width and height.

Reply:
@whimboo Could you take a look, please?